Dockerfile

```diff
@@ -1,8 +1,2 @@
-FROM registry.access.redhat.com/ubi8/openjdk-11
-USER root
-WORKDIR /tmp/src
-ADD . /tmp/src
-RUN ./gradlew assemble
-
-FROM registry.access.redhat.com/ubi8/openjdk-11
-COPY --from=0 /tmp/src/build/libs/* /deployments/
+FROM quay.io/cloudservices/rhsm-subscriptions
+USER root
```

Comment:
@lindseyburnett FYI, I noticed that the lack of an empty line at the end is causing problems with the built image on quay.

```
$ podman run -ti --rm quay.io/cloudservices/rhsm-subscriptions:pr-671-be0d04e /bin/bash
Error: unable to find user rootLABEL quay.expires-after=3d: no matching entries in passwd file
```